src/scripts/main.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const mainImg = document.getElementById('largeImg');
+
+const list = document.querySelector('#thumbs');
+
+list.addEventListener('click', (eve) => {
+  const link = event.target.closest('img');
+
+  eve.preventDefault();
+
+  if (!link) {
+    return;
+  }
+
+  const savedLink =
+    './images' + link.getAttribute('src').split('-thumb')[0] + '.png';
```

Comment:
The construction of `savedLink` assumes that the image path always contains '-thumb' and ends with '.png'. Ensure that this pattern is consistent with your actual image paths, or adjust the code to handle different patterns.